From 9a24d787f919c60c1a81241aff9fd6c6705ae44a Mon Sep 17 00:00:00 2001
From: Alex Urban <alexander.urban@ligo.org>
Date: Wed, 16 Dec 2020 15:35:29 -0600
Subject: [PATCH] Resolve circular import in setup.py

---
 setup.py | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index e9b1b0d..f545230 100644
--- a/setup.py
+++ b/setup.py
@@ -27,8 +27,6 @@ import versioneer
 
 from setuptools import setup
 
-from gwsumm.config import CONFIGDIR
-
 version = versioneer.get_version()
 cmdclass = versioneer.get_cmdclass()
 
@@ -41,8 +39,8 @@ else:
 
 # configuration files
 data_files = [
-    (CONFIGDIR.split(sys.prefix + os.path.sep).pop(),
-     glob.glob(os.path.join('share', '*.ini'))),
+    ("etc/gwsumm/configuration",
+     glob.glob(os.path.join("share", "*.ini"))),
 ]
 
 # run setup
-- 
2.21.1 (Apple Git-122.3)

